reana-ui/.eslintrc.json

```diff
@@ -14,7 +14,7 @@
     "prettier/react"
   ],
   "rules": {
-    "prettier/prettier": "error",
+    "prettier/prettier": "off", // turn of as it causes unnecessary errors
```

Comment:
We should revert this back to the original value. You can avoid CI errors by running `yarn fmt` before committing and pushing to GitHub 😄

Reply:
:see_no_evil: didn't mean to commit that, but it was showing me errors of things I didn't change myself, anyways fixed them now 😄